modules/020-cni-cilium/templates/agent/pdb.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: agent
+  namespace: d8-{{ .Chart.Name }}
+  {{- include "helm_lib_module_labels" (list . (dict "app" "agent")) | nindent 2 }}
+spec:
+  minAvailable: {{ include "helm_lib_is_ha_to_value" (list . 1 0) }}
```

Comment:
This looks like a mistake. For daemonsets you need to set the maxUnavailable option.

Reply:
Btw, HA for daemon set pods? How does this affect daemonsets?

Reply:
Done. It's... hard to keep track of all helm_lib functions.